cin_validator/ingress.py

```diff
@@ -142,11 +142,6 @@ def create_Header(self, header):
         collection_elements = ["Collection", "Year", "ReferenceDate"]
         header_dict = get_values(collection_elements, header_dict, collection_details)
 
```

Comment:
Nice. 
You can also remove the lines where they are initialized in the class' __init__() above.